assets/src/scripts/components/hydrograph/scales.js

```diff
@@ -118,7 +134,16 @@ const xScaleSelector = memoize(tsKey => createSelector(
 const yScaleSelector = createSelector(
     layoutSelector,
     visiblePointsSelector,
-    (layout, pointArrays) => createYScale(pointArrays, layout.height - (MARGIN.top + MARGIN.bottom))
+    currentVariableSelector,
+    (layout, pointArrays, currentVar) => {
+        let currentVarParmCd;
+        try {
```

Comment:
I prefer the tertiary operator here (at least I think this would work): ```currentVarParmCd = currentVarParmCd.variableCode ? currentVarParmCd.variableCode.value : null;```

Reply:
I think that would work.

Reply:
Turns out that doesn't actually work because the `currentVariableSelector` can return null, but this works: `currentVar && currentVar.variableCode ? currentVar.variableCode.value : null`.